lib/business_logic/cats_fact_bloc/content_state.dart

```diff
@@ -0,0 +1,19 @@
+part of 'content_bloc.dart';
+
+@immutable
+abstract class ContentState {}
+
+class CatsFactInitial extends ContentState {}
+
+class LoadingState extends ContentState {}
```

Comment:
fix content preffix namin(use Home or Main) and use it here. Its usable when you search Home and see all classes related to that screen.